src/components/veNft/VeNftSummaryStatsSection.tsx

```diff
@@ -0,0 +1,40 @@
+import { Trans } from '@lingui/macro'
+import { Row, Col } from 'antd'
+
+import { ThemeContext } from 'contexts/themeContext'
+import React, { useContext } from 'react'
+
+import { shadowCard } from 'constants/styles/shadowCard'
+
+interface VeNftSummaryStatsSectionProps {
+  tokenSymbolDisplayText: string
+}
+
+const VeNftSummaryStatsSection = ({
+  tokenSymbolDisplayText,
+}: VeNftSummaryStatsSectionProps) => {
+  const { theme } = useContext(ThemeContext)
+
+  return (
+    <div style={{ ...shadowCard(theme), padding: 25, marginBottom: 10 }}>
+      <h3>Staking Summary:</h3>
+      <Row>
+        <Col span={8}>
+          <p>
+            <Trans>Total staked ${tokenSymbolDisplayText}:</Trans>
+          </p>
+          <p>
+            <Trans>Total staked period:</Trans>
+          </p>
+        </Col>
+        <Col span={16}>
+          <p>0</p>
+          <p>0</p>
+        </Col>
+        <br />
```

Comment:
**suggestion**: i think we might be able to get away with removing this. If we need the padding though, wdyt about using `paddingBottom` instead?